docs/source/llms/llm-evaluate/index.rst

```diff
@@ -376,6 +376,27 @@ the score is "yes" otherwise "no".
         eval_fn=eval_fn, greater_is_better=False, name="over_10_chars"
     )
 
+If we want to create a custom metric that is dependent on other metrics, we can include those other metrics' names as an argument after ``predictions`` and ``targets``. This can be the name of a builtin metric or another custom metric.
```

Comment:
```suggestion
To create a custom metric that is dependent on other metrics, include those other metrics' names as an argument after ``predictions`` and ``targets``. This can be the name of a builtin metric or another custom metric.
```